tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -506,7 +508,7 @@ def do_rolling_bounce(self, processor, counter, current_generation):
                     self.upgraded_processors.append(processor)
 
                     # checking for the dev version which should be the only SNAPSHOT
-                    log_monitor.wait_until("Kafka version.*" + self.base_version_number + ".*SNAPSHOT",
+                    log_monitor.wait_until(kafka_version_str,
```

Comment:
does kafka_version_str already have the SNAPSHOT suffix or did something change in the naming convention on  dev? Approving pre-emptively.

Reply:
I am not 100% sure, but I assume something upstream changed and a new version is passed in.

Guess it depends what artifact are used to run the test -- if we build from release branches, the `SNAPSHOT` suffix should be contained, but if we run the test with release artifacts, the `SNAPSHOT` might have been removed already.